commands/users/banUser.js

```diff
@@ -0,0 +1,38 @@
+const { ContextMenuCommandBuilder } = require('@discordjs/builders');
+const { guildId, BATId, BALId, BAGId, BADId } = require('../../config.json');
+
+const ALL_GUILD_IDS = [guildId, BATId, BALId, BAGId, BADId];
+
+module.exports = {
+    data: new ContextMenuCommandBuilder()
```

Comment:
should we move these context commands for ban and kick to the `context` directory?